pulser-simulation/pulser_simulation/simulation.py

```diff
@@ -1156,6 +1156,9 @@ def from_sequence(
 class Simulation:
     r"""Simulation of a pulse sequence using QuTiP.
 
+    Note:
```

Comment:
Let's use this instead
```suggestion
    Warning:
```